src/main/java/io/vertx/ext/jdbc/impl/actions/JDBCSQLRowStream.java

```diff
@@ -43,6 +48,7 @@
   private final AtomicBoolean stClosed = new AtomicBoolean(false);
   private final AtomicBoolean rsClosed = new AtomicBoolean(false);
   private final AtomicBoolean more = new AtomicBoolean(false);
+  private final Deque<JsonArray> accumulator = new ArrayDeque<>(ACCUMULATOR_SIZE);
```

Comment:
I agree that it should be configurable. We cannot configure it at a query level without having to change the API so the quick win here will be passing it to the `JDBCClient` options:

```java
JDBCClient.create{Non}Shared(vertx, new JsonObject().put("rowStreamFetchSize", 100));
```

Even though it would make sense to add it to the connection itself (so it could be configured per connection) that would be a big change now that we're finalizing 3.4. The reason is that if we would do it there we would need to introduce a API change also on `sql-common` and the `async` driver.

For 3.4 we can have this just at the client level and after we can expand it to the connection level and make sure it is implemented on all `sql-common` clients.